.github/composites/docker-build-push/action.yaml

```diff
@@ -0,0 +1,129 @@
+name: 'Build and Publish Docker Image'
+description: 'Build and Publish Docker Image'
+inputs:
+  custodian_image_var:
```

Comment:
is this needed? or why is it different from docker file parameter?

Reply:
this is used for `tests/test_docker.py`

Reply:
but i can get rid of it, generate it in line just gotta change the test as well